classes-recipe/platsch.bbclass

```diff
@@ -99,6 +99,29 @@ platsch_do_compile() {
                     -strip -type TrueColor -define bmp:subtype=${SPLASH_FORMAT} bmp:- | \
                     tail -c ${SIZE_OF_RAW_BITMAP} > ${B}/${BASE}-${WIDTH}x${HEIGHT}-${SPLASH_FORMAT}.bin
                 ;;
+            XRGB8888)
+                # HACK: a quite compact way to create a bitmap in the XRGB8888 format is to
+                # let ImageMagick convert the source into its known BMP subtype XRGB8888 and
+                # to strip the BMP header by only using the final width * height * (8+8+8+8)/8
+                # bytes. For the i.MX6/i.MX8 the bitmap needs to be flipped (see [1]). This
```

Comment:
There is nothing i.MX6/8 specific here. Platsch expects the image to be top-down, as that's the natural order you would fill a dumb BO to back the DRM FB.